### DISCUSSION TITLE
137

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#70] [General]
# 潮流周刊第137期 - 九龙公园

封面图来源于尖沙咀附近尖沙咀九龙公园，里面居然还有游泳池，好多年前听过 my little airport 的粤语歌《

https://weekly.tw93.fun/posts/137-%E4%B9%9D%E9%BE%99%E5%85%AC%E5%9B%AD

**Comment by duofu** [comment]
The resource https://gw.alipayobjects.com/os/k/font/lxgwwenkaiscreenr.css was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

**Reply by tw93** [reply]
Thanks for the feedback, it's fixed!

**Comment by itcats** [comment]
寻隐：一个离线的自然语言相册搜索工具
https://queryable.app/

现在收费了，$4.99
